pkg/oci/client/config_validate.go

```diff
@@ -35,6 +36,16 @@ func validateLoadBalancerConfig(c LoadBalancerConfig, fldPath *field.Path) field
 	if c.Subnet2 == "" {
 		allErrs = append(allErrs, field.Required(fldPath.Child("subnet2"), ""))
 	}
+	if len(c.DefaultSourceCIDRs) > 0 {
+		_, err := netsets.ParseIPNets(c.DefaultSourceCIDRs...)
+		if err != nil {
+			allErrs = append(allErrs, field.Invalid(
```

Comment:
The "k8s way" of doing this error is to iterate over each one then use the `Index` function to specific which ones failed.